frontend/src/scenes/session-recordings/player/inspector/v2/PlayerInspectorList.tsx

```diff
@@ -93,10 +87,18 @@ function PlayerInspectorListItem({
                 marginBottom: PLAYER_INSPECTOR_LIST_ITEM_MARGIN / 2,
             }}
         >
-            {!isExpanded && (showIcon || windowNumber) && (
+            {!isExpanded && (
                 <div className="shrink-0 text-lg h-8 text-muted-alt flex items-center justify-center gap-1">
-                    {showIcon ? TabToIcon[item.type] : null}
-                    {windowNumber ? <IconWindow value={windowNumber} className="shrink-0" /> : null}
+                    {showIcon && tabToIconAndDescription[item.type].icon ? (
+                        <Tooltip placement="left" title={tabToIconAndDescription[item.type].tooltip}>
+                            {cloneElement(tabToIconAndDescription[item.type].icon, { width: 20, height: 20 })}
+                        </Tooltip>
+                    ) : null}
+                    {windowNumber ? (
+                        <IconWindow size="small" value={windowNumber} className="shrink-0" />
+                    ) : (
+                        <IconWindow size="small" value="A" className="shrink-0" />
```

Comment:
Personally not a fan of this. So many recordings only have 1 window which means we are showing an icon that is basically self-explantory where space is quite valuable. What led you to change this?

Reply:
I think the solution here might be to indicate them as such _only_ if there is multiple windows. Somehow indicating "we couldn't determine which window this belongs to - it could be a backend event".

Reply:
The rationale here was that the indents for events that spanned windows seemed jarring from a ux perspective, making it difficult to scan the list. Maybe the tiny A icon isn't prominent enough to differentiate these types of events either. I also don't know if having a backend event icon/tooltip could be even more confusing since we're alreayd working w/ at least 4 icons in this view. Either way I can take this out for now until we find a better solution